github/event_types.go

```diff
@@ -345,6 +351,56 @@ type PingEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// ProjectCardEvent is triggered when a project card is created, updated, moved, converted to an issue, or deleted.
+// the webhook event name is "project_card".
+//
+// GitHub docs: https://developer.github.com/v3/activity/events/types/#projectcardevent
+type ProjectCardEvent struct {
+	Action      *string      `json:"action,omitempty"`
+	Changes     *EditChange  `json:"changes,omitempty"`
+	AfterID     *int         `json:"after_id,omitempty"`
+	ProjectCard *ProjectCard `json:"project_card,omitempty"`
+
+	// The following fields are only populated by Webhook events.
+	Repo         *Repository   `json:"repository,omitempty"`
+	Org          *Organization `json:"organization,omitempty"`
+	Sender       *User         `json:"sender,omitempty"`
+	Installation *Installation `json:"installation,omitempty"`
+}
+
+// ProjectColumnEvent is triggered when a project column is created, updated, moved, or deleted.
+// the webhook event name is "project_column".
+//
+// GitHub docs https://developer.github.com/v3/activity/events/types/#projectcolumnevent
+type ProjectColumnEvent struct {
+	Action        *string        `json:"action,omitempty"`
+	Changes       *EditChange    `json:"changes,omitempty"`
+	AfterID       *int           `json:"after_id,omitempty"`
+	ProjectColumn *ProjectColumn `json:"project_column,omitempty"`
+
+	// The following fields are only populated by Webhook events.
+	Repo         *Repository   `json:"repository,omitempty"`
+	Org          *Organization `json:"organization,omitempty"`
+	Sender       *User         `json:"sender,omitempty"`
+	Installation *Installation `json:"installation,omitempty"`
+}
+
+// ProjectEvent is triggered when project is created, modified or deleted.
+// the webhook event name is "project".
+//
+// GitHub docs https://developer.github.com/v3/activity/events/types/#projectevent
+type ProjectEvent struct {
+	Action  *string     `json:"action,omitempty"`
+	Changes *EditChange `json:"changes,omitempty"`
+	Project *Project    `json:"project,omitempty"`
+
+	// The following fields are only populated by Webhook events.
+	Repo         *Repository   `json:"repository,omitempty"`
+	Org          *Organization `json:"organization,omitempty"`
+	Sender       *User         `json:"sender,omitempty"`
+	Installation *Installation `json:"installation,omitempty"`
+}
```

Comment:
Do you mind re-arranging these in the same consistent order as in `activity_events.go` and `messages.go`?

`ProjectEvent`, then `ProjectCardEvent`, then `ProjectColumnEvent`.